{"name": "Тест пополнения баланса Мегаком", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Locator.wait_for: Timeout 5000ms exceeded.\nCall log:\nwaiting for locator(\"//button [contains(text(), 'Отправить на подтверждение')]\") to be visible", "trace": "browser = <Page url='https://ibank.kicb.net/payment/provider?id=1'>\n\n    @allure.title(\"Тест пополнения баланса Мегаком\")\n    @allure.description(\"Проверка процесса пополнения баланса Мегаком\")\n    def test_mobile_top_up(browser):\n         try:\n              page = browser\n              top_up = Top_up_Page(page)\n>             top_up.mobile_top_up()\n\nProdWeb\\tests\\test_megacom.py:12: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nProdWeb\\pages\\phone_top_up_page.py:33: in mobile_top_up\n    self.page.locator(self.payment_top_up_conf_button).wait_for(timeout=5000)\nC:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\sync_api\\_generated.py:17432: in wait_for\n    self._sync(self._impl_obj.wait_for(timeout=timeout, state=state))\nC:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_locator.py:670: in wait_for\n    await self._frame.wait_for_selector(\nC:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_frame.py:323: in wait_for_selector\n    await self._channel.send(\"waitForSelector\", locals_to_params(locals()))\nC:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_connection.py:59: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x0000021EA1190A70>\ncb = <function Channel.send.<locals>.<lambda> at 0x0000021EA1131800>, is_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Locator.wait_for: Timeout 5000ms exceeded.\nE           Call log:\nE           waiting for locator(\"//button [contains(text(), 'Отправить на подтверждение')]\") to be visible\n\nC:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_connection.py:514: TimeoutError"}, "description": "Проверка процесса пополнения баланса Мегаком", "attachments": [{"name": "Ошибка", "source": "64f23eea-be37-42e6-b448-abbb6635358c-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "6f84a053-f0a2-42da-86eb-9b2cdf2314e9-attachment.txt", "type": "text/plain"}], "start": 1735102547068, "stop": 1735102553506, "uuid": "61de8203-bf58-4614-becc-8049cb12ab9e", "historyId": "f5364118b7f589cb0858bc8486ad7f1a", "testCaseId": "f5364118b7f589cb0858bc8486ad7f1a", "fullName": "tests.test_megacom#test_mobile_top_up", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_megacom"}, {"name": "host", "value": "DESKTOP-U8CRCDT"}, {"name": "thread", "value": "3760-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_megacom"}]}